### ISSUE TITLE
Loss for Training MS COCO

### ISSUE THREAD

**Issue by lliu25** [jwyang/faster-rcnn.pytorch#57]
May I ask anyone to share what the loss curve looks like? I am using the COCO dataset, and the loss is around 2.3 (around 5,000 iterations already). Is this normal? I know COCO is a larger dataset than pascal and takes longer to converge. Thanks!

**Comment by jwyang** [comment]
It seems not normal. I will share you with the loss curve.

**Comment by lliu25** [comment]
@jwyang Thank you so much. BTW I was using VGG16 ask the CNN. Also, any plan to release the trained model for VGG16 on COCO? Thank you again.

**Comment by McDo** [comment]
@jwyang would you please post your coco loss curve here? thank you.

**Comment by Noebarrell** [comment]
@jwyang thank you so much for your amazing code. I was hoping you could share the voc and coco loss curves with me as well please, am wondering about convergence during training
